modules/viewing-registered-models.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+
+[id="viewing-registered-models_{context}"]
+= Viewing registered models
+
+[role='_abstract']
+You can view the details of models registered in {productname-short}, such as registered versions, deployments, and metadata associated with the model.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* An available model registry exists in your deployment, and contains at least 1 registered model.
+* You have access to the model registry that contains the model that you want to view.
+
+.Procedure
+. From the {productname-short} dashboard, click *Model Registry*.
+. From the *Model registry* dropdown menu, select the model registry that contains the model that you want to view.
+. The *Model Registry* page provides a high-level view of registered models, including the model name, labels, last modified timestamp, and owner of each model.
+. Use the search bar to find a model in the list. You can search with a keyword by default, or click the search bar drop-down menu and select *Owner* to search by model owner.
+. Click the name of a model to view more details.
++
+The details page for the model opens.
+
+.. On the *Versions* tab, you can view registered versions of the model.
+.. On the *Details* tab, you can view the description, labels, custom properties, model ID, owner, and *Last modified* and *Created* timestamps for the model.
```

Comment:
Minor suggestion to either bold all UI field names or none (also the UI I'm looking at says **Last modified at** and **Created at**)

Reply:
Same as previous - thank you!